src/Microsoft.ML.Recommender/RecommenderCatalog.cs

```diff
@@ -80,6 +80,8 @@ internal RecommendationTrainers(RecommendationCatalog catalog)
             /// <para>The basic idea of matrix factorization is finding two low-rank factor matrices to apporimate the training matrix.</para>
```

Comment:
>apporimate [](start = 104, length = 10)

approximate

Reply:
please add spell-checker plugin, if you haven't already
[https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellChecker](https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellChecker)

---
In reply to: [276862518](https://github.com/dotnet/machinelearning/pull/3409#discussion_r276862518) [](ancestors = 276862518)

Reply:
Super helpful tip. Thanks.

---
In reply to: [276869057](https://github.com/dotnet/machinelearning/pull/3409#discussion_r276869057) [](ancestors = 276869057,276862518)